bytestring_splitter/__init__.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, *message_parameters):
     def __call__(self, splittable, return_remainder=False, msgpack_remainder=False):
 
         if not self.is_variable_length:
-            if not any((return_remainder, msgpack_remainder)) and len(self) != len(splittable):
+             if not (return_remainder or msgpack_remainder) and len(self) != len(splittable):
```

Comment:
I don't know what happens with GH suggestions but it added a white space here that wasn't in the suggestion (I'm sure about it). I don't care about the attribution, btw, so feel free to fix this as it's best for the PR history.